src/plugins/mozprefs/README.md

```diff
@@ -40,28 +40,32 @@ In Mozilla preference files `.` is used to separate sections, while elektra uses
 will all result in `lockPref("a.lock.key", "lock");`
 
 ## Example ##
-
-    % kdb mount prefs.js user/prefs mozprefs
-    % kdb setmeta user/prefs/lock/a/lock/key type boolean
-    % kdb set user/prefs/lock/a/lock/key true
-    % kdb setmeta user/prefs/pref/a/default/key type string
-    % kdb set user/prefs/pref/a/default/key "i'm a default key"
-    % kdb setmeta user/prefs/user/a/user/key type integer
-    % kdb set user/prefs/user/a/user/key 123
-
-    % kdb export user/prefs
-
-    [lock/a/lock]
-    key = true
-    [pref/a/default]
-    key = i'm a default key
-    [user/a/user]
-    key = 123
-
-
-    % cat `kdb file user/prefs`
-
-    lockPref("a.lock.key", true);
-    pref("a.default.key", "i'm a default key");
-    user_pref("a.user.key", 123);
-
+```sh
+# Backup-and-Restor:/examples/examples/prefs
```

Comment:
Typo: Backup-and-Restor**e**. Why is examples twice?
Can you make `/examples` as default for Backup-and-Restore?